src/core/i18n/utils.js

```diff
@@ -244,7 +244,15 @@ export function makeI18n(i18nData: I18nConfig, lang: string, _Jed: Jed = Jed) {
   // can type-check all the components that rely on the i18n object.
   // Note: the available locales for tests are controlled in tests/setup.js
   // $FLOW_IGNORE
-  i18n.formatNumber = (number) => new Intl.NumberFormat(lang).format(number);
+  i18n.formatNumber = (number) => {
+    if (Intl && Intl.NumberFormat) {
```

Comment:
Yeah, I worry less about redefining `undefined`... but more just about `Intl` being an undefined object in the global scope. Sometimes that makes things error weirdly.

We need to use `typeof Foo !== 'undefined'` elsewhere for `window` so I'd stick to it here. If we wanna change it we can do it app-wide in the future. It's just that fails the server tests, IIRC.